LanguageFeatures/Private-named-parameters/static_semantics_A05_t01.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2025, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Given a named initializing formal or field parameter (for a
+/// primary constructor) with private name `p` in constructor `C`:
+/// ...
+/// If there is no error then:
+/// ...
+/// - The instance variable initialized by the parameter (and declared by it, if
+///   the parameter is a field parameter), has the private name `p`.
+///
+/// @description Check that the instance variable initialized by the
+/// initializing formal parameter has the private name `p`.
```

Comment:
Good point! It would actually be possible to do everything right, except that the actual argument passed to `p: arg` is never assigned to the correct instance variable `_p`.

This should be true for dynamic invocations (of tear-offs, because they are constructors). Do we cover that already?

Reply:
Sounds good!